src/services/stringCompletions.ts

```diff
@@ -448,10 +448,27 @@ namespace ts.Completions.StringCompletions {
 
         fragment = ensureTrailingDirectorySeparator(fragment);
 
-        // const absolutePath = normalizeAndPreserveTrailingSlash(isRootedDiskPath(fragment) ? fragment : combinePaths(scriptPath, fragment)); // TODO(rbuckton): should use resolvePaths
         const absolutePath = resolvePath(scriptPath, fragment);
         const baseDirectory = hasTrailingDirectorySeparator(absolutePath) ? absolutePath : getDirectoryPath(absolutePath);
 
+        // check for a version redirect
+        const packageJsonPath = findPackageJson(baseDirectory, host);
+        if (packageJsonPath) {
+            const packageJson = readJson(packageJsonPath, host as { readFile: (filename: string) => string | undefined });
+            const typesVersions = (packageJson as any).typesVersions;
+            if (typeof typesVersions === "object") {
+                const versionResult = getPackageJsonTypesVersionsPaths(typesVersions);
+                const versionPaths = versionResult && versionResult.paths;

```

Comment:
```suggestion
                const versionPaths = versionResult?.paths;
```

Reply:
You might want to hide whitespace changes; you’re requesting changes on a lot of code I didn’t write 😑

Reply:
The whitespace-ignoring diff isn't that magical, these aren't in the same spot :D

In any case, we can make the changes now.